src/main/java/com/zufar/onlinestore/payment/enums/PaymentStatus.java

```diff
@@ -1,12 +1,37 @@
 package com.zufar.onlinestore.payment.enums;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
 public enum PaymentStatus {
-    REQUIRES_PAYMENT_METHOD,
-    REQUIRES_CONFIRMATION,
-    REQUIRES_CAPTURE,
-    REQUIRES_ACTION,
-    PROCESSING,
-    CANCELED,
-    SUCCEEDED
+
+    PAYMENT_METHOD_IS_REQUIRED("payment_intent.requires_payment_method",
+            "Payment method is required"),
+
+    CONFIRMATION_IS_REQUIRED("payment_intent.requires_confirmation",
+            "Payment confirmation is required"),
+
+    CAPTURE_IS_REQUIRED("payment_intent.requires_capture",
```

Comment:
`PAYMENT_CAPTURE_IS_REQUIRED` is more clear

Reply:
Done